### ISSUE TITLE
🚀 - `InputInline` multiline

### ISSUE THREAD

**Issue by waterplea** [taiga-family/taiga-ui#8982]
### Description

We need a textarea based version input inline so it can be multiline. We also need to be able to place actions like that:

<img width="1338" alt="image" src="https://github.com/user-attachments/assets/0243777e-9a85-4165-af1e-a9d4c7c2f052">

**Comment by splincode** [comment]
@waterplea something like?

```html
<input tuiInputInline />
```

```html
<textarea tuiInputInline></textarea>
```

```html
<div contenteditable tuiInputInline [(ngModel)]="value"></div>
```

**Comment by waterplea** [comment]
Just the textarea, for contenteditable we have editor

**Comment by 40oleg** [comment]
I would like to take it.
Is this behavior similar to what you mean?
https://github.com/user-attachments/assets/f4f4ea47-43d6-4be9-8814-91b77a4acc8d

**Comment by 40oleg** [comment]
https://github.com/user-attachments/assets/d8d3ee1c-0053-479e-a1cf-b47b4019dc92

**Comment by waterplea** [comment]
@40oleg yes, that looks correct. Also keep in mind the pencil icon should not be a part of the component itself, people should just be able to easily place and hide it, like with current inline single line input.
